app/locale/af/LC_MESSAGES/django.po

```diff
@@ -795,11 +795,11 @@ msgstr ""
 msgid "Failed to obtain search result for the phone number."
 msgstr "Kon nie soekresultaat vir die foonnommer kry nie."
 
-msgid "Family name"
-msgstr "Van"
+msgid "Family name (Surname)"
+msgstr ""
```

Comment:
It looks like this change deletes the msgstrs for name label translations for every language besides en and ml. Is that expected?

Reply:
Yes, that's the default behavior by rerunning tools/update_messages because messages are keyed by the English text. They will be retranslated in the next translation cycle.

I manually updated Malayalam translation as I wrote in the pull request description.

Reply:
I see, ok.